saleor/plugins/user_email/plugin.py

```diff
@@ -22,63 +29,61 @@
-    account_password_reset_template: str
-
-
-event_map = {
-    UserNotifyEvent.ACCOUNT_CONFIRMATION: send_account_confirmation,
-    UserNotifyEvent.ACCOUNT_SET_CUSTOMER_PASSWORD: send_account_set_customer_password,
-    UserNotifyEvent.ACCOUNT_DELETE: send_account_delete,
-    UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_CONFIRM: send_account_change_email_confirm,
-    UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_REQUEST: send_account_change_email_request,
-    UserNotifyEvent.ACCOUNT_PASSWORD_RESET: send_account_password_reset_event,
-    UserNotifyEvent.INVOICE_READY: send_invoice,
-    UserNotifyEvent.ORDER_CONFIRMATION: send_order_confirmation,
-    UserNotifyEvent.ORDER_FULFILLMENT_CONFIRMATION: send_fulfillment_confirmation,
-    UserNotifyEvent.ORDER_FULFILLMENT_UPDATE: send_fulfillment_update,
-    UserNotifyEvent.ORDER_PAYMENT_CONFIRMATION: send_payment_confirmation,
-    UserNotifyEvent.ORDER_CANCELED: send_order_canceled,
-    UserNotifyEvent.ORDER_REFUND_CONFIRMATION: send_order_refund,
-}
+def get_user_event_map():
+    return {
+        UserNotifyEvent.ACCOUNT_CONFIRMATION: send_account_confirmation,
+        UserNotifyEvent.ACCOUNT_SET_CUSTOMER_PASSWORD: (
+            send_account_set_customer_password
+        ),
+        UserNotifyEvent.ACCOUNT_DELETE: send_account_delete,
+        UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_CONFIRM: send_account_change_email_confirm,
+        UserNotifyEvent.ACCOUNT_CHANGE_EMAIL_REQUEST: send_account_change_email_request,
+        UserNotifyEvent.ACCOUNT_PASSWORD_RESET: send_account_password_reset_event,
+        UserNotifyEvent.INVOICE_READY: send_invoice,
+        UserNotifyEvent.ORDER_CONFIRMATION: send_order_confirmation,
+        UserNotifyEvent.ORDER_FULFILLMENT_CONFIRMATION: send_fulfillment_confirmation,
+        UserNotifyEvent.ORDER_FULFILLMENT_UPDATE: send_fulfillment_update,
+        UserNotifyEvent.ORDER_PAYMENT_CONFIRMATION: send_payment_confirmation,
+        UserNotifyEvent.ORDER_CANCELED: send_order_canceled,
+        UserNotifyEvent.ORDER_REFUND_CONFIRMATION: send_order_refund,
+    }
 
 
 class UserEmailPlugin(BasePlugin):
     PLUGIN_ID = "mirumee.notifications.user_email"
     PLUGIN_NAME = "User emails"
     # TODO the configuration will be implemented in separate pull request
 
-    # DEFAULT_CONFIGURATION = [
-    #     {"name": "account-confirmation_template", "value": None},
-    #     {"name": "account-password-reset-template", "value": None},
-    # ]
-    #
-    # CONFIG_STRUCTURE = {
-    #     "account-confirmation_template": {
-    #         "type": ConfigurationTypeField.SECRET,
-    #         "help_text": "",
-    #         "label": "Account confirmation template",
-    #     },
-    #     "account-password-reset-template": {
-    #         "type": ConfigurationTypeField.SECRET,
-    #         "help_text": "",
-    #         "label": "Account password reset template",
-    #     },
-    # }
+    DEFAULT_CONFIGURATION = [] + DEFAULT_EMAIL_CONFIGURATION  # type: ignore
```

Comment:
`DEFAULT_EMAIL_CONFIGURATION` is a list already, why do we need `[] +`?

Reply:
We want to create a new list here, not copy the reference of the DEFAULT one.  In next PRs this list will not be empty :D